stable/kong/README.md

```diff
@@ -67,10 +67,15 @@ and their default values.
 | admin.ingress.hosts            | List of ingress hosts.                                                           | `[]`                |
 | admin.ingress.path             | Ingress path.                                                                    | `/`                 |
 | admin.ingress.annotations      | Ingress annotations. See documentation for your ingress controller for details   | `{}`                |
-| proxy.useTLS                   | Secure Proxy traffic                                                             | `true`              |
-| proxy.servicePort              | TCP port on which the Kong Proxy Service is exposed                              | `8443`              |
-| proxy.containerPort            | TCP port on which the Kong app listens for Proxy traffic                         | `8443`              |
-| proxy.nodePort                 | Node port when service type is `NodePort`                                        |                     |
+| proxy.useOnlyTLS               | Use only TLS for the proxy port                                                  | true                |
```

Comment:
Please update these to reflect the new changes

Reply:
Updated.